kanopy-platform/argoslower
# Istio 1.21 upgrade

Updates istio dependencies to v1.21.6, which is the [latest v1.21.x release](https://github.com/istio/api/tags?after=v1.22.6)

No open [dependabot alerts](https://github.com/kanopy-platform/argoslower/security/dependabot?q=is%3Aopen)

### Testing
1. Start kind cluster, `make kind-istio`, enable [argoEvents](https://github.com/10gen/platform/blob/master/data/cluster/kind/argo.yaml#L3) and `kanopy apply kind --context=kind-kind --skip-controllers=namespace --module=argoEvents`
2. Run `skaffold dev` from this repo
3.
```
kubectl apply -f examples/testers/gw.yaml -f examples/testers/event-bus.yaml -f examples/testers/demo-secret.yaml
sleep 2
kubectl apply -f examples/testers/es-jira.yaml
```
4. See that VirtualService and AuthorizationPolicies are created
```
kubectl get vs -n routing-rules devops-argo-tester-jira
kubectl get authorizationpolicy -n routing devops-argo-tester-jira
```
5. 
```
kubectl -n devops logs -f argo-tester-jira-eventsource...
```
6.
```
curl -k -XPOST  -H 'Authorization: Bearer thisisanexample' --data '{ "msg": "hello-world" }' https://webhooks.example.com/devops/argo-tester-jira/example
```
7. See that eventsource pod logs got the message
